lib/events.ts

```diff
@@ -44,7 +43,11 @@ const getMixpanel = _.once((balenaUrl: string) => {
  * information in Sentry.io error reporting, for CLI debugging purposes
  * (mainly unexpected/unhandled exceptions -- see also `lib/errors.ts`).
  */
-export function trackCommand(commandSignature: string) {
+export async function trackCommand(commandSignature: string) {
+	const Sentry = await import('@sentry/node');
```

Comment:
By the way, `@sentry/node` could perhaps be bumped to v5.15.5 (current latest):
https://github.com/getsentry/sentry-javascript/blob/master/CHANGELOG.md
I have not tested it, though.